db/controller/controller.js

```diff
@@ -27,17 +30,52 @@ exports.getArticlesById = (req, res, next) => {
     });
 };
 
+exports.getCommentsByArticleId = (req, res, next) => {
+  const id = req.params.article_id;
+  fetchCommentsByArticleId(id)
+    .then((article) => {
+      res.status(200).send(article);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.postComment = (req, res, next) => {
+  const id = req.params.article_id
+  const newComment = req.body;
+  addNewComment(id, newComment)
+    .then((comment) => {
+      res.status(201).send(comment);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.removeComment = (req, res, next) => {
+  const commentToRemove = req.params.comment_id
+  deletingComment(commentToRemove)
+    .then((deletedComment) => {
+      res.status(204).send(deletedComment);
```

Comment:
You don't need to send anything back as Express will not send anything back as we have used a 204 status code